ProjectCoimbra.UWP/Project.Coimbra/Pages/CreateAccountPage.xaml.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed under the MIT License.
+
+namespace Coimbra.Pages
+{
+    using DataAccessLibrary;
+    using Windows.UI.Xaml;
+    using Windows.UI.Xaml.Controls;
+    using Windows.UI.Xaml.Media.Animation;
+
+    /// <summary>
+    /// The Signup / Create Account Page
+    /// </summary>
+    public sealed partial class CreateAccountPage : Page
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CreateAccountPage"/> class.
+        /// </summary>
+        public CreateAccountPage()
+        {
+            this.InitializeComponent();
+        }
+
+        /// <summary>
+        /// Handles the Click event of the CreateAccountButton control.
+        /// </summary>
+        /// <param name="sender">The source of the event.</param>
+        /// <param name="e">The <see cref="RoutedEventArgs"/> instance containing the event data.</param>
```

Comment:
IMO, this method is clear enough without comment. The same for the method below.